include/rootfs.mk

```diff
@@ -47,7 +47,7 @@ TARGET_DIR_ORIG := $(TARGET_ROOTFS_DIR)/root.orig-$(BOARD)
 
 ifdef CONFIG_CLEAN_IPKG
   define clean_ipkg
-	-find $(1)/usr/lib/opkg/info -type f -and -not -name '*.control' -delete
+	-find $(1)/usr/lib/opkg/info -type f '!' -name '*.control' | $(XARGS) $(RM)
```

Comment:
`-delete` is also available on FreeBSD, so this might be readily available by fiat.